src/confluent_kafka/src/Consumer.c

```diff
@@ -1113,13 +1200,33 @@ static PyMethodDef Consumer_methods[] = {
           "  :raises: RuntimeError if called on a closed consumer\n"
 	  "\n"
 	},
+	{ "incremental_assign", (PyCFunction)Consumer_incremental_assign, METH_O,
+	  ".. py:function:: incremental_assign(partitions)\n"
+	  "\n"
+	  "  Incrementally add the provided list of :py:class:`TopicPartition` "
+          "to the current partition assignment.\n"
```

Comment:
Maybe beef this up a bit more with the constraints, e.g., no dups, etc.
Same for incr_unassign

Reply:
yep, absolutely should.